packages/terra-application-links/examples/test-examples/ApplicationTabsDefault.example.jsx

```diff
@@ -11,7 +11,7 @@ export default () => (
       initialEntries={testLinkConfig.map(link => link.path)}
       initialIndex={0}
     >
-      <div style={{ border: '1px solid lightGray', width: '100%', backgroundColor: 'gray', height: '39px', position: 'relative' }}>
+      <div style={{ border: '1px solid lightGray', width: '100%', backgroundColor: 'green', height: '39px', position: 'relative' }}>
```

Comment:
Curious, any reason for the example background color change?

Reply:
Contrast ratio between the gray background and the text wouldn't pass accessibility tests. :(